jgt_tools/run_tests.py

```diff
@@ -16,14 +16,15 @@
 __doc__ = __doc__.format("\n    ".join(__commands_to_run))
 
 
-def run_tests(do_setup=False, self_check=False, verbose=True):
+def run_tests(do_setup=False, self_check=False, verbose=True, additional_args=""):
     """Run code checks."""
     if do_setup:
         env_setup(verbose)
     if self_check:
         run_self_check(verbose=verbose)
 
-    execute_command_list(__commands_to_run, verbose=verbose)
+    commands_to_run = [f"{x} {' '.join(additional_args)}" for x in __commands_to_run]
```

Comment:
I went ahead and went with the first one because (a) it does follow best practices for default parameters and (b) we aren't using `jgt_common` here and didn't want to pull that in for one use. As always, I'm open to alternate thoughts.